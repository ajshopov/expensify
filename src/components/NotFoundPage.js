import React from "react";
import { Link } from "react-router-dom";

const NotFoundPage = () => (
  <div>
    This is NotFoundPage.
    <Link to="/">Go Home</Link>
  </div>
);

export default NotFoundPage;