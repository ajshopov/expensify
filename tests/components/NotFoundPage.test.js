import React from "react";
import { shallow } from "enzyme";
import NotFoundPage from "../../src/components/NotFoundPage";

test('should render 404 page', () => {
  const wrapper = shallow(<NotFoundPage />)
  expect(wrapper).toMatchSnapshot()
})
