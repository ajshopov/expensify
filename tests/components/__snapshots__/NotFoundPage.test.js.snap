// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 404 page 1`] = `
<div>
  This is NotFoundPage.
  <Link
    to="/"
  >
    Go Home
  </Link>
</div>
`;
